### ISSUE TITLE
Loop thru visualizations

### ISSUE THREAD

**Issue by kylerobert425** [naztronaut/dancyPi-audio-reactive-led#62]
Is there a way to run a single script to automatically change the visualization type every few minutes? Ive tried a few things but as soon as the visualization is running, I can only quit it with a keyboard interrupt so I'm not sure how to stop it and start another one via a single script

**Comment by rekk2** [comment]
you could crontab -e but I'm having the problem of having to restart every time I turn it off

**Comment by kylerobert425** [comment]
Yeah thats my solution for now but it requires the pi to get restarted to start over
